src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/FrameworkExtensionTest.php

```diff
@@ -527,7 +527,7 @@ private function checkAssetsPackages(ContainerBuilder $container, $legacy = fals
         $package = $container->getDefinition($packages['bar']);
         $this->assertUrlPackage($container, $package, array('https://bar2.example.com'), $legacy ? null : 'SomeVersionScheme', $legacy ? '%%s?%%s' : '%%s?version=%%s');
 
-        $this->assertEquals($legacy ? 'assets.empty_version_strategy' : 'assets._version__default', (string) $container->getDefinition('assets._package_bar')->getArgument(1));
+        $this->assertEquals('assets.empty_version_strategy', (string) $container->getDefinition('assets._package_bar')->getArgument(1));
```

Comment:
I see. 

This behaviour was introduced in [this commit](https://github.com/symfony/symfony/pull/16511/files)

If I'm right, the problem is with [this condition change](https://github.com/symfony/symfony/pull/16511/files#diff-0e793081ceb720201745c982a568903fL592), because `$package['version']` key is always exists, since [it has a null default value](https://github.com/symfony/symfony/pull/16511/files#diff-5e7347edce37c5886ec67b7ba02f3a9cR491). So `$version` never will be equals with `$defaultVersion` (which is a Reference with an id of `'assets._version__default'`)